examples/TypeScriptMessaging/App.tsx

```diff
@@ -23,6 +23,7 @@ import {
 import { useStreamChatTheme } from './useStreamChatTheme';
 import { GestureHandlerRootView } from 'react-native-gesture-handler';
 import { useFlipper } from 'stream-chat-react-native-devtools';
+import { StreamChatRN } from 'package/src/utils/getResizedImageUrl';
```

Comment:
very weird path 😿 

`package/src/utils/getResizedImageUrl`

Reply:
let us think through implementing this.. as this is once done cant be changed after

the import path is ideally the below options

* import StreamChatRN  from 'stream-chat-react-native'
* import { setConfig, setUIComponents } from 'stream-chat-react-native'

Reply:
`import StreamChatRN from 'stream-chat-react-native'
is the case, it was left there from some internal testing, let me change that